test/dlc_tests/sanity/test_pre_release.py

```diff
@@ -471,8 +470,8 @@ def _run_dependency_check_test(image, ec2_connection):
 @pytest.mark.canary("Run dependency tests regularly on production images")
 @pytest.mark.parametrize("ec2_instance_type", ["c5.4xlarge"], indirect=True)
 @pytest.mark.skipif(
-    (is_canary_context() and not is_time_for_canary_safety_scan()),
-    reason="Executing test in canaries pipeline during only a limited period of time.",
+    is_canary_context(),
+    reason="Skipping the test on canaries.",
```

Comment:
you can just remove `@pytest.mark.canary` if you are removing this from canaries, and then also remove these skip conditions

Reply:
Done, thanks!!